match/lib/match/utils.rb

```diff
@@ -1,10 +1,21 @@
 module Match
   class Utils
     def self.import(item_path, keychain)
-      command = "security import #{item_path.shellescape} -k ~/Library/Keychains/#{keychain.shellescape}"
+      # Existing code expects that a keychain name will be expanded into a default path to Libary/Keychains
+      # in the user's home directory. However, this will not allow the user to pass an absolute path
+      # for the keychain value
+      keychain_path = File.join(Dir.home, 'Library', 'Keychains', keychain)
+
+      # So, if the passed value can't be resolved as a file in Library/Keychains, just use it as-is
+      # as the keychain path.
+      keychain_path = keychain unless File.exist?(keychain_path)
+
+      command = "security import #{item_path.shellescape} -k #{keychain_path.shellescape}"
       command << " -T /usr/bin/codesign" # to not be asked for permission when running a tool like `gym`
       command << " -T /usr/bin/security"
-      command << "&> /dev/null" # we couldn't care less about the output
+      command << " &> /dev/null" unless $verbose
+
+      return command if Helper.test?
```

Comment:
I know we have this across the code base sometimes, but I think we can use an `expect` for `Helper.backticks` to make it cleaner and have a higher test coverage (as `Helper.backticks` doesn't get executed in tests currently)